task-2/src/store/reducers/moviesReducer.js

```diff
@@ -1,4 +1,9 @@
-import { SET_MOVIES, ADD_MOVIE, DELETE_MOVIE } from '../actionTypes';
+import {
+	SET_MOVIES,
+	// ADD_MOVIE,
```

Comment:
Consider going through the codebase and either delete commented code or leave a TODO comment explaining why it's still there.

Reply:
resolved.